utils/monitoring.js

```diff
@@ -0,0 +1,88 @@
+const { errors } = require('arsenal');
+const promClient = require('prom-client');
+const { Registry } = require('prom-client');
+const { http } = require('httpagent');
+const CountMaster = require('../CountItems/CountMaster');
+
+const aggregatorRegistry = new promClient.AggregatorRegistry();
+const { collectDefaultMetrics } = promClient;
+
+
+// Histogram of the bucket processing duration, by the utilization service.
+const bucketProcessingDuration = new promClient.Histogram({
+    name: 's3_countitems_bucket_listing_duration_seconds',
+    help: 'Bucket processing duration',
+    buckets: [1, 10, 60, 600, 3600, 18000, 36000],
+});
+
+const consolidationDuration = new promClient.Histogram({
+    name: 's3_countitems_bucket_merge_duration_seconds',
+    help: 'Duration of metrics consolidation in seconds',
+    buckets: [0.01, 0.05, 0.1, 0.2, 0.5, 1, 2, 5, 10],
+});
+
+const workersCount = new promClient.Counter({
```

Comment:
is it `workers` or `buckets` ?

Reply:
renamed to buckets indeed this was leftover when I updated the description : https://github.com/scality/s3utils/pull/312/commits/094b51c921935a4def339cbd48b61b2f8cf40a3d#diff-612a4c10e405510cdaad879589d4a7d157df6e0ff42b4645d2a21d4d17e41fa9R24